docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -313,6 +215,97 @@ In addition to using `fetch()`, there's a other ways to query the API:
 </div>
 </div>
 
+<a id="accessing-the-graphql-playground" class="devdocsAnchor"></a>
+
+## Accessing the GraphQL Playground
+
+To access the GraphQL Storefront API Playground<sup>1</sup> and documentation:
+
+1. Login to a BigCommerce store enrolled in the beta
+2. Navigate to **Advanced Settings** > **Storefront API Playground**<sup>2</sup>
+
+The GraphQL Storefront API Playground will be opened:
+
+![GraphQL Storefront API Playground](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-storefront-api-playground.png "GraphQL Storefront API Playground")
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+    
+<!-- theme: info -->
+
+### Note
+
+> 1. GraphQL Playground is a GraphQL IDE built on Electron. For more information, see [GraphQL Playground](https://electronjs.org/apps/graphql-playground) on [electrongjs.org](https://electronjs.org)
+
+> 2. If the **Storefront API Playground** link is not visible, the store is not enrolled in the Beta program. To enroll, [contact support](https://support.bigcommerce.com/SubmitCase) (all stores using Stencil are now eligible).

```

Comment:
Suggest sticking to one convention for list objects: punctuated end or unpunctuated